### ISSUE TITLE
How to set the number of search results?

### ISSUE THREAD

**Issue by mcyt** [Jackett/Jackett#3849]
search results show 100 each time
<item>
*
</item>
help me.

**Comment by mcyt** [comment]
Limit the number, by the number of pages, show

**Comment by mcyt** [comment]
win Jackett Version: 0.10.198.0

**Comment by garfield69** [comment]
the search results number is deliberately aimed to be around 100 results, but is mostly dependent on what each site returns by default in the first page of their search response.
some sites allow users to specify more results per page in their user profiles, so that's the first place to start looking.
 for other sites with fixed results per page, that allow specifying a page number on the search request, we can code the jackett indexer to fetch additional pages during the search request.
but we try not to fetch more than 100 results to avoid overloading the site with traffic. its the polite thing to do. 
some site don't like bots fetching results and you don't want to loose access.
for other sites that don't easily allow more results per page and where we cant easily code additional page requests, then you are stuck with what you get on the first search page the site presents.

is there a particular site you want more results from? it could be that the indexer needs reviewing/improvement if possible, to return more results (up to 100 is the goal).

**Comment by kaso17** [comment]
you can try to increase the limit returned by jackett by adding `&limit=1000` to the RSS/torznab url.
But as @garfield69 already explained many indexers will only return the results from the first page.

**Comment by mcyt** [comment]
> the search results number is deliberately aimed to be around 100 results, but is mostly dependent on what each site returns by default in the first page of their search response.
> some sites allow users to specify more results per page in their user profiles, so that's the first place to start looking.
> for other sites with fixed results per page, that allow specifying a page number on the search request, we can code the jackett indexer to fetch additional pages during the search request.
> but we try not to fetch more than 100 results to avoid overloading the site with traffic. its the polite thing to do.
> some site don't like bots fetching results and you don't want to loose access.
> for other sites that don't easily allow more results per page and where we cant easily code additional page requests, then you are stuck with what you get on the first search page the site presents.
> 
> is there a particular site you want more results from? it could be that the indexer needs reviewing/improvement if possible, to return more results (up to 100 is the goal).

Yeah, I know. I just want to know about this feature, used to limit the number. Usually used to limit the latest 5-10, and occasionally need more than 100
Thank you for the reply.
